tests/test_kubernetes.py

```
@@ -2315,3 +2323,201 @@ def test_close_cleanup_error(
+        session = SandboxKubernetesSession()
+        stdout_chunks: list[str] = []
+
+        mock_resp = MagicMock()
+        mock_resp.is_open.side_effect = [True, False]
+        mock_resp.peek_stdout.return_value = True
+        mock_resp.read_stdout.return_value = "chunk1"
+        mock_resp.peek_stderr.return_value = False
+        mock_resp.returncode = 0
+
+        with patch("llm_sandbox.kubernetes.stream", return_value=mock_resp):
+            exit_code, (stdout, stderr) = session.container_api.execute_command(
```

Ruff (0.15.1)

[warning] Unpacked variable `exit_code` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)